pkgs/games/shattered-pixel-dungeon/generic.nix

```diff
@@ -55,6 +55,9 @@ let
 
   depsPath' = if depsPath != null then depsPath else ./. + "/${pname}/deps.json";
 
+  # "Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0."
+  gradle = gradle_8;
+
```

Comment:
Does it apply to all shattered-pixel-dungeon variants?

Reply:
Yes, it does, from what I can tell.